src/assets/scripts/client/trafficGenerator/SpawnPatternModel.js

```diff
@@ -489,6 +489,7 @@ export default class SpawnPatternModel extends BaseModel {
         this.speed = this._extractSpeedFromJson(spawnPatternJson);
         this.method = spawnPatternJson.method;
         this.rate = parseFloat(spawnPatternJson.rate);
+        this.entrail = _get(spawnPatternJson, 'entrail', this.entrail);
```

Comment:
seems simple enough.  does this actually fix the `Surge` spawnPattern`s?

Reply:
The surge spawnPatterns are broken? :open_mouth:

Reply:
probably?

Reply:
broken indeed. We have #156 for this.